locale/en/LC_MESSAGES/django.po

```diff
@@ -1832,15 +2179,64 @@ msgid ""
 "instructions for setting a new one."
 msgstr ""
 
+#: temp/temp_ve/core/templates/versions.html:8
```

Comment:
@sewagodimo please remove these sentences here as well this template is meant or admins as well